src/main/scala/battleship/routes/SimpleRoutes.scala

```diff
@@ -1,10 +1,15 @@
 package battleship.routes
 
+
```

Comment:
Not really necessary to include this class in the commit, since there are only white space changes